sctk/src/event_loop/mod.rs

```diff
@@ -292,6 +292,8 @@ where
         F: FnMut(IcedSctkEvent<T>, &SctkState<T>, &mut ControlFlow),
     {
         let mut control_flow = ControlFlow::Poll;
+        let mut cursor_position = PhysicalPosition::new(0, 0);
+        let mut scale_factor = 1.0;
```

Comment:
Do you think this value should be tracked per-surface? I wonder if it could be problematic for an application with multiple windows spread over outputs with different scales.

Reply:
Done. I've tested with moving a window across displays with different scales.